examples/sources/CustomSource.md

```diff
@@ -0,0 +1,24 @@
+# How to add a spout as a custom source
```

Comment:
Rename the file to README.md so it will show the doc in the github itself.

Reply:
Will do